client/src/pages/Home.css

```diff
@@ -252,15 +161,11 @@ form {
 	margin: 10px;
 }
 
-.questionsDiv {
+.questionsDiv {	
 	border: 1px solid black;
-	width: 80%;
+	width: 95%;
 	margin: 0 auto;
-	position: center;
-	min-height: 400px;
-	margin-bottom: 20px;
```

Comment:
Please, look at the margin botton as there is static footer. There might be issues if there is no margin bottom.